### PR TITLE
Log post

### PR DESCRIPTION
**Pull request by felisity09**
Wrote an express middleware to store log into the database

### CODE REVIEW

File: server/controllers/logController.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const Logs = require('../models/logs');
+
+const logController = {};
+
+//store a single log
+logController.storeLog = async (req, res, next) => {
+  try {
+    console.log('storeLog');
+    const { log } = req.body;
+    // TODO: able to store more than one log
```
Comment (Agfeynman) [inline server/controllers/logController.js:10]: Pls remove the comment. We need a different controller for that

File: server/controllers/logController.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    if (!log) {
+      console.log('missing log');
```
Comment (Agfeynman) [inline server/controllers/logController.js:12]: You want to use the logger for log the error server side and the error handler to send the information to the client.

File: server/routes/v1.routes.js [hunk 3]
```diff
@@ -1,2 +1,16 @@
...
+
+const logController = require('../controllers/logController');
+
+Router.post('/log/store', logController.storeLog, (req, res, next) => {
+  const { logStored } = res.locals;
+
+  return res.status(201).json({
+    status: true,
+    log: logStored,
+    message: 'storeLog middleware: Successfully stored log into DB',
```
Comment (Agfeynman) [inline server/routes/v1.routes.js:12]: Can you remove the storeLog middleware from here. The client doesn't want to know that detail implementation.

**Review by Agfeynman — CHANGES_REQUESTED** [review]
Nice job Felicity!!!. I made some suggestions. Also you need to remove all the 'working' commit messages and put a semantic message instead. That's important in case we have some problem in the future and we want to go back or if we want to create a changelog using a tool which is probable.

**Review by velveteen-ant — APPROVED** [review]
approving -- I saw working test during live share session. I agree on also using more descriptive commit messages

**Pull request merged**